README.md

```diff
@@ -1381,6 +1381,45 @@ print(url)
 
 </details>
 
+#### Get comments of a video
+You can use a Comments class for that.
+```python
+from youtubesearchpython import *
+
+# You can either pass an ID or a URL
+video_id = "_ZdsmLgCVdU"
+comments = Comments(video_id)
+
+print(f'Comments Retrieved: {len(comments.comments["result"])}')
+
+while comments.hasMoreComments:
+    print('Getting more comments...')
+    comments.getNextComments()
+    print(f'Comments Retrieved: {len(comments.comments["result"])}')
+
+print('Found all the comments.')
+```
+
+<details>
+ <summary> Example Result</summary>
+
+```bash
+Videos Retrieved: 20
```

Comment:
You probably forgot to change the output of the example here (same for async version).
Also, it would be great to add an example of what it actually returns (in JSON).